modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -0,0 +1,116 @@
+= Mule Upgrade Tool
+
+*Latest version: 1.0.0*
+
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool.adoc[tag=intro]
+
+The Mule upgrade tool runs in any environment that is capable of running a standalone Mule instance. This tool can update only standalone Mule instances that are running any Mule 4 version. You can upgrade local Mule instances to
+//the latest available version or to
+a particular Mule distribution that you have previously downloaded and unpacked. You can also use the Mule upgrade tool to update Mule instances located in different folders.
+
+As part of the upgrade process, the Mule upgrade tool also updates the Runtime Manager agent version if it is installed in your current Mule instance.
+
+== Before You Begin
+
+Before starting your upgrade, ensure that you have:
+
+* A currently operational Mule 4 instance, in _stopped_ status to prepare for the upgrade
+* The Mule upgrade tool inside your current Mule instance's `$MULE_HOME/tools` directory
++
+If the Mule upgrade tool is not bundled with your current Mule distribution, download the tool from the https://help.mulesoft.com/s/[Support Portal^] and copy it into the `$MULE_HOME/tools` directory of your current Mule instance.
+* The new Mule runtime engine distribution you want to upgrade to
```

Comment:
I'm not sure if we should refer to "the distribution you want to upgrade to" or just refer to "the latest distribution", as the scope and aim of the upgrade tool is that you upgrade to the latest one. @estebanwasinger what do you think?